model/bootstrap/node_info_test.go

```diff
@@ -9,14 +9,28 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/onflow/flow-go/model/bootstrap"
-	"github.com/onflow/flow-go/model/flow/order"
+	"github.com/onflow/flow-go/model/flow"
 	"github.com/onflow/flow-go/utils/unittest"
 )
 
-func TestSort(t *testing.T) {
+func TestIdentityListCanonical(t *testing.T) {
 	nodes := unittest.NodeInfosFixture(20)
-	nodes = bootstrap.Sort(nodes, order.Canonical)
-	require.True(t, bootstrap.ToIdentityList(nodes).Sorted(order.Canonical))
+	// make sure the list is not sorted
+	nodes[0].NodeID[0], nodes[1].NodeID[0] = 2, 1
+	require.False(t, flow.IsIdentifierCanonical(nodes[0].NodeID, nodes[1].NodeID))
+
+	ids := bootstrap.ToIdentityList(nodes)
+	assert.False(t, flow.IsIdentityListCanonical(ids))
+
+	nodes = bootstrap.Sort(nodes, flow.Canonical)
```

Comment:
can we make sure that original list has not been modified in this test?

Reply:
Good point!